specs/experimental/gov-delegation.md

```diff
@@ -309,17 +305,20 @@ enum AllowanceType {
 | `Absolute`  | `0`    | The amount of votes delegated is fixed and denominated in the `GovernanceToken`'s decimals  |
 | `Relative`  | `1`    | The amount of votes delegated is relative and denominated in percentages.                   |
 
-## Backwards Compatibility
-
-The `GovernanceDelegation` contract ensures backwards compatibility by allowing the migration of delegation state from the
-`GovernanceToken`.
-
 ## Migration
 
 All write functions in the `GovernanceDelegation` MUST check if the users interacting with it have been migrated by
 checking the `migrated` mapping from its [storage](#storage). If a user has not been migrated, the `GovernanceDelegation`
 MUST copy the delegation and checkpoint data from the token contract to its own state. After copying the data, the
-`GovernanceDelegation` MUST update the `migrated` mapping to reflect that the address has been migrated.
+`GovernanceDelegation` MUST update the `migrated` mapping to reflect that the address has been migrated, and remove the
```

Comment:
I actually don't think we need the `migrated` mapping anymore? Since we're not forwarding stuff to the original contract.

Reply:
yeah the `GovernanceDelegation` contract doesn't really need it, but it could be a helper for external contracts like governor: if `migrated` is true, get data from `GovernanceDelegation`, otherwise from `GovernanceToken`. 

if these contracts check `delegates[_addr]` only in `GovernanceToken`, they won't be able to know if there's a delegation in `GovernanceDelegation`, so they will always have to call both.

Reply:
Ah since we maintain the invariant that you can't have a delegation in both contracts then what you can do is just check if `delegates[_addr]` inside `GovernanceToken` is `address(0)`

Reply:
agreed, updated in https://github.com/ethereum-optimism/specs/pull/342/commits/eee936c9321706d7939f83103eb6246aa4b6e4f0